app.js

```diff
@@ -0,0 +1,149 @@
+function shipPlacementGeneration()
+{
+    createGrid(getContainer("player-grid-container"), true);
+    createGrid(getContainer("computer-grid-container"), false);
+}
+
+function createGrid(container, isUser)
+{
+    for(let i = 0; i < 10; i++)
+    {
+        for(let j = 0; j < 10; j++)
+        {
+            var item = document.createElement("div");
+            if(isUser)
+            {
+                item.id = "user";
+                item.id += " square-" + i + "-" + j;
```

Comment:
Move class name generation to the separate method
function getShipPartClass(x, y) { return `square-${x}-${y}`; }